classes/gpm.php

```diff
@@ -27,6 +27,7 @@ public static function GPM()
     {
         if (!static::$GPM) {
             static::$GPM = new GravGPM();
+            static::$GPM->loadRemoteGrav();
```

Comment:
As a quick way to make this 'safer' and backward compatible, this could be to wrap it in a `method_exists` check.  Something like:

```
if (method_exists('GravGPM', 'loadRemoteGrav') {
  static::$GPM->loadRemoteGrav();
} else {
  static::$GPM = new GravGPM();
}
```